webpack/webpack.config.client.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const common = require('./webpack.config.common');
+
+const isDevMod = process.env.NODE_ENV === 'development';
+
+module.exports = merge(common, {
+  name: 'client',
+
+  target: 'web',
+
+  entry: [
+    isDevMod && 'webpack-hot-middleware/client',
+    './src/client.jsx',
+  ].filter(Boolean),
+
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        include: /src/,
+        use: [
+          isDevMod ? 'style-loader' : MiniCssExtractPlugin.loader,
```

Comment:
I'd rather separate this config to 2 files in order to have more flexible configuration